driver/heap.go

```diff
@@ -0,0 +1,33 @@
+// This file contains code from these Go 1.14.5 source files:
+// - container/heap/example_intheap_test.go
+//
+// That code is covered by the following copyright notice,
+// and the license file mentioned is available at:
+// https://github.com/golang/go/blob/go1.14.5/LICENSE
```

Comment:
Is a reference to the license file sufficient? The file itself says:
>Redistributions of source code must retain the above copyright
>notice, this list of conditions and the following disclaimer.
And referring doesn't seem like quite the same thing as retaining.